app/components/Typography.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Text, TextStyle } from 'react-native';
+
+import { useLanguageDirection } from '../locales/languages';
+
+import { Typography as TypographyStyles } from '../styles';
+
+type TypographicUse = 'headline1' | 'headline2' | 'body1' | 'body2' | 'body3';
+
+interface TypographyProps {
+  use?: TypographicUse;
```

Comment:
ideally we could remove this prop and just use 'style' everywhere as this is the convention in RN and is one less thing to need to maintain be understood by onboarding developers. We didnt remove it here as to reduce the amount of git/merge conflict churn.